src/utils/patcher.ts

```diff
@@ -74,7 +74,7 @@ class PatchInfo<T, R, A extends any[]> {
 
     private error(patch: Patch<T, R, A>, type: "PrePatch" | "PostPatch", error: any) {
         const message =
-            (patch.plugin ? `[${patch.plugin}] ` : "") +
+            (patch?.plugin ? `[${patch.plugin}] ` : "") +
```

Comment:
? patch should never be undefined

Reply:
it is though

Reply:
thats what crashes the settings page

Reply:
fix the actual problem then instead of doing this, patch shouldn't be undefined